src/CommonDCModelDropdown.php

```diff
@@ -272,7 +272,7 @@ private function isCellFilled(array $cell, int $orientation, int $hpos, float $d
                     array_reverse($cell[$hpos]) // If orientation is rear, reverse the array
                     : $cell[$hpos],
                 0,
-                $depth * 4
+                ((int) ceil($depth)) * 4
```

Comment:
Unsure about this one, see comment here: https://github.com/glpi-project/glpi/pull/15448#discussion_r2227902623.

My suggestion send the correct type but I don't know if the feature works as expected here.